### ISSUE TITLE
Data Logging Script Exporting Different Scenarios to Their Own .tsv Files

### ISSUE THREAD

**Issue by robbieorvis** [Energy-Innovation/eps-us#96]
When running the data logging script, Vensim/python is creating a separate .tsv file for each scenario. I think in the past there used to be a single file with all scenario results (if not, this would be a nice feature).

**Comment by jrissman** [comment]
No, the data logging script has never put all scenarios' results into a single .tsv file.  It always has produced one .tsv file for each scenario that is run (each of which is defined by its own .cin file).  This way, each .tsv file is associated with its own .cin file, which is probably clearest and easiest for most users to understand.  It also facilitates pasting the different scenarios' data into different, identically-formatted tabs of a spreadsheet for easy cross-scenario comparisons.  Wouldn't putting them all into one file make things less convenient, rather than more?

**Comment by jrissman** [comment]
I could, of course, make it a switch you can toggle in the script itself, if we want to support both behaviors.

**Comment by robbieorvis** [comment]
I think it depends. Generally speaking, when I'm using this tool, I'm comparing multiple scenarios in a single file, so I end up having to copy and paste the data from the separate .tsv files into a single spreadsheet. So for my purposes, it would be easiest of all the data were in a single file and organized by scenario. I don't see a lot of scenarios in which having separate .tsv files makes things easier, provided the scenario name is listed if they are reported out in a single file.

**Comment by jrissman** [comment]
Okay, got it.  This is doable and can be included in the next EPS version.
